image: /home/tim/code/ros/final_project_ws/install/reinforcement_planning/share/reinforcement_planning/config/map.pgm
mode: trinary
resolution: 0.05
origin: [-1.21, -2.39, 0]
negate: 0
occupied_thresh: 0.65
free_thresh: 0.25